### PR TITLE
fix: Added whitespace trimming for api and domain flags in project:create

### PR DESCRIPTION
**Pull request by CalemRoelofsSB**
Small fix for project:create flags

## Proposed Changes

  - As caught by @MateuszPol in testing, added regex to remove whitespace in --api and --domain flags to ensure they get parsed correctly

  - Update erroneous example string in project:member:list

### CODE REVIEW

**Review by barryadk — DISMISSED** [review]
Nice enhancement.

File: test/support/parse-input.test.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const {
   getDomainIdentifierArg,
   getProjectIdentifierArg,
   readConfigFile,
-  reqType 
+  reqType, splitFlagParams
```
Comment (barryadk) [inline test/support/parse-input.test.js:10]: To keep the format consistent - could you add it on a new line?

File: src/support/command/parse-input.js [hunk 2]
```diff
@@ -70,7 +70,7 @@ const readConfigFile = filename => {
 
 const splitPathParams = path => path.split('/').filter(Boolean)
 
-const splitFlagParams = flag => flag.split(',').filter(Boolean)
+const splitFlagParams = flag => flag.replace(/\s+/g, '').split(',').filter(Boolean)
```
Comment (barryadk) [inline src/support/command/parse-input.js:73]: Could you split, then trim()?

File: test/support/parse-input.test.js (hunk 1 above)
Comment (CalemRoelofsSB) [reply test/support/parse-input.test.js:10]: Automatically imported it when I wrote the test 🤦  I'll change it now

File: src/support/command/parse-input.js (hunk 2 above)
Comment (CalemRoelofsSB) [reply src/support/command/parse-input.js:73]: Probably would be a bit more readable for the next person who looks at it, I'll change it to that

**Review by barryadk — APPROVED** [review]

**Pull request merged**